controllers/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -64,7 +64,7 @@ const (
 	hostOperatorTag        statusComponentTag = "hostOperator"
 	memberConnectionsTag   statusComponentTag = "members"
 	counterTag             statusComponentTag = "MasterUserRecord and UserAccount counter"
-	minutesAfterUnready    time.Duration      = 10
+	minutesAfterUnready    time.Duration      = 10 * time.Minute
```

Comment:
Let's rename it. It's not the number of minutes actually. It's a duration.
durationAfterUnready?

Reply:
done in fd6f00a